velox/exec/Aggregate.h

```diff
@@ -190,6 +208,17 @@ class Aggregate {
     return numNulls_ && (group[nullByte_] & nullMask_);
   }
 
+  void incrementRowSize(char* row, uint64_t bytes) {
+    VELOX_DCHECK(rowSizeOffset_);
+    uint32_t* ptr = reinterpret_cast<uint32_t*>(row + rowSizeOffset_);
+    uint64_t size = *ptr + bytes;
+    *ptr = std::min<uint64_t>(bytes, std::numeric_limits<uint32_t>::max());
+  }
+
+  void setNull(char* group) {
```

Comment:
This method appears unused. What is it for? Can it be added together with first usage?